examples/jre/src/main/java/com/fiskaly/sdk/demo/jre/Main.java

```diff
@@ -2,13 +2,297 @@
 
 import com.fiskaly.sdk.*;
 
+import java.io.IOException;
+import java.util.Collections;
+import java.util.UUID;
+
+import com.fiskaly.sdk.factories.GsonFactory;
+import com.google.gson.Gson;
+
 public class Main {
+  static String tssUUID = "";
```

Comment:
Thanks, I do actually use that type when creating the UUIDs, but then store the new UUID as a string — probably just because this is the way the Swift code does it, but maybe it also makes it easier to append to strings to make request URLs, without calling toString every time. But maybe toString is called implicitly when appending a UUID to a string; I will experiment with that.

Reply:
I just changed it to store the UUIDs as UUIDs.